appveyor.yaml

```diff
@@ -0,0 +1,22 @@
+version: '{build}'
+branches:
+  only:
+  - master
+configuration: RelWithDebInfo
+build_script:
+- ps: >-
+    choco install cmake
+    choco install python2
+    mkdir build
+    cd build
+    cmake -D ENABLE_TRACE=ON -D BUILD_CLAR=ON -D STDCALL=ON .. -G"Visual Studio 11"
+    cmake --build . --config RelWithDebInfo
+test_script:
+- ps: ctest -V .
+notifications:
+- provider: Email
+  to:
+  - emeric.fermas@gmail.com
```

Comment:
Shouldn't we try and configure the build result routing to IRC, CampFire, .... ? See http://www.appveyor.com/docs/notifications